library/oneview_id_pools_ipv4_subnet.py

```diff
@@ -109,7 +109,7 @@ def execute_module(self):
         self.data['type'] = self.data.get('type', 'Subnet')
 
         if self.state == 'present':
-            return self.resource_present(resource, 'id_pools_ipv4_subnets')
+            return self.resource_present(resource, 'id_pools_ipv4_subnet')
```

Comment:
The fact name when you are using a normal module is in singular, different from the facts module where it should be in plural.